tests/test_util.py

```diff
@@ -323,3 +323,22 @@ def test_unfulfilled_kwargs(self):
         exc = pytest.raises(ValueError, check_callable_args, func, [1], {})
         assert str(exc.value) == ('The following keyword-only arguments have not been supplied in '
                                   'kwargs: y')
+
+
+def not_a_coro(x):
+    pass
+
+
+async def a_coro(x):
+    pass
+
+
+class TestIsCoroutineFunctionPartial(object):
+    def test_non_coro(self):
+        assert iscoroutinefunction_partial(not_a_coro) is False
```

Comment:
Replace with `assert not ...` as per PEP 8.

Reply:
Amended.